views/doc.show.html.twig

```diff
@@ -7,18 +7,23 @@
 {% block content %}
 
     <div id="searchbar" class="clearfix">
-      <input id="docsearch" type="text" placeholder="Search documentation">
+        <input id="docsearch" type="text" placeholder="Type here to search documentation..." role="searchbox"
+               title="Enter text here and submit to search the documentation"/>
     </div>
 
     {% if toc|length %}
         <ul class="toc">
             {{ macros.tocTree(toc|length > 3 ? toc : toc.1) }}
         </ul>
     {% endif %}
+
     {{ doc|raw }}
 
     <p class="fork-and-edit">
-        Found a typo? Something is wrong in this documentation? <a href="https://github.com/composer/composer/edit/master/doc/{{ file }}">Fork and edit</a> it!
+        Found a typo? Something is wrong in this documentation? <a
```

Comment:
This wrapping does not make sense.

Reply:
no it does not (at least not in templates). You had to re-wrap all templates you edited due to this assumption. This seems to make it clear to me that it was a wrong assumption.

Reply:
Ah crap. Indeed. Part of it looked so. In that case this will make wrapping much easier. :D